app/src/main/java/com/codesoom/assignment/application/ToyService.java

```diff
@@ -0,0 +1,19 @@
+package com.codesoom.assignment.application;
+
+import java.util.List;
+
+/**
+ * 장난감 관리 서비스를 제공하는 인터페이스이다.
+ */
+public interface ToyService <T>{
```

Comment:
띄어쓰기에 대한 리뷰를 남기고 싶지는 않지만...
```suggestion
public interface ToyService<T> {
```

작업을 커밋하기 전에 포매터를 실행해 주시면 도움이 될 거에요.